nexus/types/src/deployment/planning_input.rs

```diff
@@ -384,6 +410,18 @@ impl PlanningInput {
         self.sleds.get(sled_id).map(|details| &details.resources)
     }
 
+    pub fn omicron_zone_external_ips(
+        &self,
+    ) -> &BTreeMap<OmicronZoneUuid, OmicronZoneExternalIp> {
+        &self.omicron_zone_external_ips
+    }
+
```

Comment:
This appears unused?

Reply:
Indeed, this and the below one was part of the half-done code in #5552. No reason to keep it around for now. Removed.